Makefile

```diff
@@ -118,6 +118,11 @@ lcov:
 	$(call run-cmake-release, -DBUILD_TESTS=TRUE -DBUILD_LCOV=TRUE)
 	ctest --test-dir build/release/test --output-on-failure -j ${TEST_JOBS}
 
+extension-lcov:
```

Comment:
I think you can the lcov for extenison in another pr.
Before enabling the code coverage for extension, we should add more extension tests.